web/src/beta/pages/Editor/index.tsx

```diff
@@ -1,38 +1,28 @@
-import { useEffect, useState } from "react";
-import { useLocation, useParams } from "react-router-dom";
+import { useParams } from "react-router-dom";
 
 import LeftPanel from "@reearth/beta/features/LeftPanel";
 import Navbar, { Tab } from "@reearth/beta/features/Navbar";
 import RightPanel from "@reearth/beta/features/RightPanel";
 import Visualizer from "@reearth/beta/features/Visualizer";
+import NotFound from "@reearth/classic/components/atoms/NotFound";
 import { styled } from "@reearth/services/theme";
 
 type Props = {};
 
-const tabs = ["scene", "story", "widgets", "publish"];
+function isTab(tab: string): tab is Tab {
+  return ["scene", "story", "widgets", "publish"].includes(tab);
```

Comment:
We already have the ["scene, "story"....] array in navigationHooks as Tab. Since it doesn't seem to make complete sense there, can you make a types.ts file inside src/beta/ and move the Tab type to that file? 🙏🏻 Then you can import it in to the navigationHooks and this file from there